### PR TITLE
Issue 1/lat lon inputs

### PR DESCRIPTION
**Pull request by jjfrench**
#1 Allow user to specify which datasets to use for lat/lon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ must be supplied for every input):
   a logical name representing such a DOI (see
   [Specifying a DOI](#specifying-a-doi))
 
+- `coord-type`: Coordiante type in reference to the longitude/latitude datasets.
```
Comment (jjfrench) [inline README.md:43]: ope, *coords-type

File: README.md [hunk 2]
```diff
@@ -40,6 +40,9 @@ must be supplied for every input):
...
+   Types such as `lowestmode` and `highestreturn` correlating to (`lat_lowestmode`, `lon_lowestmode`) and (`lat_highestreturn`,`lon_highestreturn`) respectively.
```
Comment (chuckwondo) [inline README.md:44]:
```suggestion
- `coords-type`: Coordinates type specifying the latitude/longitude datasets.
   The algorithm adds the prefixes `"lat_"` and `"lon_"` to the coordinates type
   to determine the names of the datasets to use for the coordinates. For
   example, specifying a coordinates type of `lowestmode` will cause the
   algorithm to use the `lat_lowestmode` and `lon_lowestmode` datasets for
   latitude and longitude, respectively.
```

File: src/gedi_subset/subset.py [hunk 3]
```diff
@@ -192,6 +202,13 @@ def main(
         CMRHost.maap,
         help="CMR hostname",
     ),
+    coords_type: str = typer.Option(
+        ...,
+        help=(
+            "Coordinate type in reference to longitude/latitude"
+            " such as values `lowestmode` and `highestreturn`."
```
Comment (chuckwondo) [inline src/gedi_subset/subset.py:209]:
```suggestion
            "Coordinates type specifying the latitude/longitude datasets. The"
            " prefixes 'lat_' and 'lon_' are added to this value to determine the"
            " dataset names for latitude and longitude, respectively. For example,"
            " specifying 'lowestmode' will cause the 'lat_lowestmode' and"
            " 'lon_lowestmode' datasets to be used for latitude and longitude,"
            " respectively."
```

**Review by chuckwondo — APPROVED** [review]
Awesome! Just a couple of minor suggestions for clarifying the description of the coordinates type (plus 1 typo correction).

**Review by chuckwondo — APPROVED** [review]
Looks great!

**Pull request merged**